models/Order.js

```diff
@@ -0,0 +1,34 @@
+var keystone = require('keystone'),
+  Types = keystone.Field.Types;
+
+/**
+ * Order Model
+ * ==========
+ */
+
+var Order = new keystone.List('Order', {
+  map: { name: 'user' },
+  autokey: { path: 'slug', from: 'user', unique: true }
```

Comment:
Use the user's name and the release as the autokey's value creation scheme (the from attribute)